tests/test_playbooks/search_facts.yml

```diff
@@ -13,6 +13,11 @@
     vars:
       resource: subnets
       return_length: 0
+  - include: tasks/search_facts.yml
+    vars:
+      resource: organizations
+      full_details: true
+      return_length: 2
```

Comment:
Is this `2` a good test result? I think it depends a lot on your test setup. And you did not even include the fixtures for organization, so i guess, 'Test Organization' isn't even one of those.